cfme/tests/automate/custom_button/test_service_objects.py

```diff
@@ -713,3 +716,67 @@ def test_custom_button_open_url_service_obj(objects, button_group):
         1550002
     """
     pass
+
+
+@pytest.fixture(params=["Service", "Other"], scope="module")
+def unassigned_btn_setup(request, appliance, provider, service):
+    if request.param == "Service":
+        btn_type = request.param
+        obj = MyService(appliance, name=service.name)
+        destinations = [ViaUI, ViaSSUI]
+    else:
+        btn_type = "Provider"
+        obj = provider
+        destinations = [ViaUI]
+
+    gp = appliance.collections.button_groups.instantiate(
+        text="[Unassigned Buttons]", hover="Unassigned buttons", type=btn_type
+    )
+    yield obj, gp, destinations
+
+
+@pytest.mark.provider([VMwareProvider], override=True, scope="module", selector=ONE)
+def test_custom_button_unassigned_behavior_objs(
+    appliance, setup_provider, unassigned_btn_setup, request
+):
+    """ Test unassigned custom button behavior
+
+    Note: Service unassigned custom button should display on SSUI but not OPS UI.
+    For other than service objects also fallows same behaviour l.e. not display on OPS UI.
+
+    Polarion:
+        assignee: ndhandre
+        initialEstimate: 1/6h
+        caseimportance: medium
+        caseposneg: positive
+        testtype: functional
+        startsin: 5.8
+        casecomponent: CustomButton
+        testSteps:
+            1. Create unassigned custom button on service and one other custom button object.
+            2. Check destinations OPS UI should not display unassigned button but SSUI should.
+
+    Bugzilla:
+        1653195
+    """
+
+    obj, gp, destinations = unassigned_btn_setup
+
+    with appliance.context.use(ViaUI):
+        button = gp.buttons.create(
+            text="btn_{}".format(fauxfactory.gen_alphanumeric(3)),
+            hover="btn_hover_{}".format(fauxfactory.gen_alphanumeric(3)),
+            system="Request",
+            request="InspectMe",
+        )
+        assert button.exists
+        request.addfinalizer(button.delete_if_exists)
```

Comment:
I am not sure but I am worried that the button wont't be deleted in the correct context.